dashboard/continuous-profiling.md

```diff
@@ -21,6 +21,8 @@ summary: TiDB Dashboard 持续性能分析功能 (Continuous Profiling)
 
 - 该功能适用于使用 v1.9.0 及以上版本 TiUP 或 v1.3.0 及以上版本 TiDB Operator 部署和升级的集群，不支持二进制包部署和升级的集群。
 
+- 在 TiKV 和 TiFlash 实例中有偶发性失败，将会导致实例重启。
```

Comment:
How about moving this into the "WARNING" section above?

Reply:
Done